app/models/course/discussion/topic.rb

```diff
@@ -10,6 +10,17 @@ class Course::Discussion::Topic < ActiveRecord::Base
 
   accepts_nested_attributes_for :posts
 
+  # Topics to be displayed in the comments center.
+  scope :to_display, (lambda do
```

Comment:
@allenwq then maybe the scope now should be called `displayed_globally` or something.